spec/02-integration/04-admin_api/02-apis_routes_spec.lua

```diff
@@ -87,6 +88,54 @@ describe("Admin API #" .. kong_config.database, function()
       end)
 
       describe("errors", function()
+        it("returns 415 on invalid content-type", function()
+          local res = assert(client:request {
+            method = "POST",
+            path = "/apis",
+            body = '{"hello": "world"}',
+            headers = {["Content-Type"] = "invalid"}
+          })
+          assert.res_status(415, res)
+        end)
+        it("returns 415 on missing content-type with body ", function()
+          local res = assert(client:request {
+            method = "POST",
+            path = "/apis",
+            body = "invalid"
+          })
+          assert.res_status(415, res)
+        end)
+        it("returns 400 on missing body with application/json", function()
+          local res = assert(client:request {
+            method = "POST",
+            path = "/apis",
+            headers = {["Content-Type"] = "application/json"}
+          })
+          assert.res_status(400, res)
```

Comment:
When we are returning a `400` error here, I think a second assertion for the body should be added for explicitness, ensuring that it is a schema error from our DAO (and not a Lapis error or some other sort of error).